publisher/sql/sanitize.sql

```diff
@@ -152,5 +195,16 @@ from (select
                                                     and c.table_schema = st.schemaname and c.table_name = st.relname)
       where pgd.description ilike '%SANITIZE_AS_JSONB%') as res;
 
+-- call radically_sanitize_jsonb on every column containing SANITIZE_AS_EMPTY_JSONB in the comment
+select sanitizing.radically_sanitize_jsonb(res.table_name :: varchar, res.column_name :: varchar)
```

Comment:
can we call it similarly to comment, like `sanitize_as_empty_jsonb`?

Reply:
done

Reply:
not pushed?